### DISCUSSION TITLE
关于客户端上报方式

### DISCUSSION THREAD

**Discussion by mnpq-top** [zdz/ServerStatus-Rust#164] [Q&A]
起因📰 cloudflare面板看到每个客户端每天有大概90k条请求。

疑问❓客户端上报地址是https://example.com/report，是走的grpc协议吗？

我尝试了用nginx反代grpc端口，不起用。
ServerStatus:1.7.0
nginx:1.22.1

nginx listen 2053,location /{grpc_pass grpc://__}

如果不是能否支持呢？

**Comment by zdz** [comment]
1. 每秒上报一次，一天就是这个量级
2. 走 http+pb 协议
3. http 已经够用，grpc 可以自行研究
